kong/llm/drivers/anthropic.lua

```diff
@@ -209,7 +209,9 @@ function _M.subrequest(body, conf, http_opts, return_res_table)
     error("body must be table or string")
   end
 
-  local url = fmt(
+  -- may be overridden
+  local url = (conf.model.options and conf.model.options.upstream_url)
+    or fmt(
```

Comment:
If `ai_shared.upstream_url_format[DRIVER_NAME]` and `ai_shared.operation_map[DRIVER_NAME][conf.route_type].path` are guaranteed to be non-nil, then string concatenation will